.eslintrc

```diff
@@ -1,2 +1,3 @@
 ---
 extends: standard
+parser:  "babel-eslint"
```

Comment:
why do we need this? seems to work fine without it for me

Reply:
is one of the class features not supported by regular eslint right now?

Reply:
I added `eslint-babel` to support the static attribute. and I think the class syntax is much cleaner.
I fixed this now by remove the static attribute and work with old way (static).

```js
MyClass.MyStaticAttribute = "MyValue"
```

Reply:
interesting that eslint still doesn't support it...